.travis.yml

```diff
@@ -6,14 +6,16 @@ before_install:
   - wget -q https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
   - export MINICONDA_PREFIX="$HOME/miniconda"
   - bash miniconda.sh -b -p $MINICONDA_PREFIX
+  - source "$HOME/miniconda/etc/profile.d/conda.sh"
+  - hash -r
```

Comment:
Is this line necessary?

```bash
help hash
hash: hash [-lr] [-p pathname] [-dt] [name ...]
    For each NAME, the full pathname of the command is determined and
    remembered.  If the -p option is supplied, PATHNAME is used as the
    full pathname of NAME, and no path search is performed.  The -r
    option causes the shell to forget all remembered locations.  The -d
    option causes the shell to forget the remembered location of each NAME.
    If the -t option is supplied the full pathname to which each NAME
    corresponds is printed.  If multiple NAME arguments are supplied with
    -t, the NAME is printed before the hashed full pathname.  The -l option
    causes output to be displayed in a format that may be reused as input.
    If no arguments are given, information about remembered commands is displayed.
```

If it isn't, please remove (that help text above makes it sound like something we don't need, but, I'm not sure). If it is, can you just make a note of why it's needed, or how things fail if it isn't included?

Reply:
That line clears the shell's pathname cache. It is probably not needed, but was recommended in the conda example, presumably because conda is doing interesting things with the shell's env variables. I've dropped it in my most recent PR, and it will stay dropped unless it breaks something. : )